### PR TITLE
Remove ubuntu1804

### PR DESCRIPTION
**Pull request by jburel**
Remove ref to ubuntu18.04

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,8 +3,6 @@ ansible/*.retry
 ansible/.DS_Store
 **/omeroweb-install-*
 **/centos7*/**
```
Comment (dominikl) [inline .gitignore:5]: I guess centos7 could be removed too from gitignore.

Comment (dominikl) [inline .gitignore:5]: I guess centos7 could be removed too from gitignore
```suggestion
```

Comment (omereleases) [reply .gitignore:5]: centos7 has not been remove yet

Comment (jburel) [reply .gitignore:5]: centos7 has not been removed yet

Comment (dominikl) [reply .gitignore:5]: Oh ok 👍

**Pull request merged**